deploy/charts/templates/daemonset.yaml

```
@@ -0,0 +1,162 @@
+{{- if .Values.ndm.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "openebs-ndm.fullname" . }}
+  {{- if .Values.ndm.annotations }}
```

This can be shortened to make it more readable like this:

```suggestion
  {{- with .Values.ndm.annotations }}
  annotations: {{ toYaml . | nindent 4 }}
  {{- end }}
```
See https://banzaicloud.com/blog/creating-helm-charts-part-2/

same goes for other places.

---

done